soroban-env-host/src/budget.rs

```diff
@@ -111,33 +33,75 @@ impl CostType {
 /// The parameters for a `CostModel` are calibrated empirically. See this crate's
 /// benchmarks for more details.
 
-#[derive(Default, Debug, Clone, PartialEq, Eq, PartialOrd, Ord, Hash)]
-pub struct CostModel {
-    pub(crate) const_param: u64,
-    pub(crate) lin_param: u64,
+// #[derive(Default, Debug, Clone, PartialEq, Eq, PartialOrd, Ord, Hash)]
```

Comment:
If this commented-out code is dead I think it should be removed.

Reply:
Oops sorry I forgot to clean it up. 
Deleted.

Reply:
thanks!